workflow/rules/ref.smk

```diff
@@ -101,3 +101,14 @@ rule update_lineages:
         "logs/{date}/pangolin/update.log",
     shell:
         "(mkdir -p {output} && wget -qO- https://github.com/cov-lineages/lineages/archive/master.tar.gz | tar xvz --strip-components=1 -C {output})> {log} 2>&1"
+
+
+rule get_gisaid_provision:
+    output:
+        temp("resources/gisaid/provision.json"),
+    params:
+        user_and_pw=os.environ["GISAID_API_TOKEN"],
+    log:
+        "logs/get_gisaid_provision.log",
+    shell:
+        "(curl -u {params.user_and_pw} https://www.epicov.org/epi3/3p/resseq02/export/provision.json.xz | xz -d -T0 > {output})> {log} 2>&1"
```

Comment:
Just use $GISAID_API_TOKEN directly here. If you still have user_and_pw in params, it is leaked into the report and snakemake's metadata storage.